src/config/backends/raspberry-pi.ts

```diff
@@ -0,0 +1,157 @@
+	private static forbiddenConfigKeys = [
+		'disable_commandline_tags',
+		'cmdline',
+		'kernel',
+		'kernel_address',
+		'kernel_old',
+		'ramfsfile',
+		'ramfsaddr',
+		'initramfs',
+		'device_tree_address',
+		'init_emmc_clock',
+		'avoid_safe_mode',
+	];
+
+	public matches(deviceType: string): boolean {
+		return _.startsWith(deviceType, 'raspberry') || deviceType === 'fincm3';
+	}
+
+	public async getBootConfig(): Promise<ConfigOptions> {
+		let configContents = '';
+
+		if (await fs.exists(RPiConfigBackend.bootConfigPath)) {
+			configContents = await fs.readFile(
+				RPiConfigBackend.bootConfigPath,
+				'utf-8',
+			);
+		} else {
+			await fs.writeFile(RPiConfigBackend.bootConfigPath, '');
+		}
+
+		const conf: ConfigOptions = {};
+		const configStatements = configContents.split(/\r?\n/);
+
+		for (const configStr of configStatements) {
+			// Don't show warnings for comments and empty lines
+			const trimmed = _.trimStart(configStr);
+			if (_.startsWith(trimmed, '#') || trimmed === '') {
+				continue;
+			}
+			let keyValue = /^([^=]+)=(.*)$/.exec(configStr);
+			if (keyValue != null) {
+				const [, key, value] = keyValue;
+				if (!_.includes(RPiConfigBackend.arrayConfigKeys, key)) {
+					conf[key] = value;
+				} else {
+					if (conf[key] == null) {
+						conf[key] = [];
+					}
+					const confArr = conf[key];
+					if (!_.isArray(confArr)) {
+						throw new Error(
+							`Expected '${key}' to have a config array but got ${typeof confArr}`,
+						);
+					}
+					confArr.push(value);
+				}
+				continue;
+			}
+
+			// Try the next regex instead
+			keyValue = /^(initramfs) (.+)/.exec(configStr);
+			if (keyValue != null) {
+				const [, key, value] = keyValue;
+				conf[key] = value;
+			} else {
+				log.warn(`Could not parse config.txt entry: ${configStr}. Ignoring.`);
+			}
+		}
+
+		return conf;
+	}
+
+	public async setBootConfig(opts: ConfigOptions): Promise<void> {
+		let confStatements: string[] = [];
+
+		_.each(opts, (value, key) => {
+			if (key === 'initramfs') {
+				confStatements.push(`${key} ${value}`);
+			} else if (_.isArray(value)) {
+				confStatements = confStatements.concat(
+					_.map(value, (entry) => `${key}=${entry}`),
+				);
+			} else {
+				confStatements.push(`${key}=${value}`);
+			}
+		});
+
+		const confStr = `${confStatements.join('\n')}\n`;
+
+		await remountAndWriteAtomic(RPiConfigBackend.bootConfigPath, confStr);
+	}
+
+	public isSupportedConfig(configName: string): boolean {
+		return !_.includes(RPiConfigBackend.forbiddenConfigKeys, configName);
+	}
+
+	public isBootConfigVar(envVar: string): boolean {
+		return _.startsWith(envVar, RPiConfigBackend.bootConfigVarPrefix);
+	}
+
+	public processConfigVarName(envVar: string): string {
+		return envVar.replace(RPiConfigBackend.bootConfigVarRegex, '$2');
+	}
+
+	public processConfigVarValue(key: string, value: string): string | string[] {
+		if (_.includes(RPiConfigBackend.arrayConfigKeys, key)) {
+			if (!_.startsWith(value, '"')) {
```

Comment:
If we know it's a string then we can use the native version
```suggestion
			if (!value.startsWith('"')) {
```
- the same goes for the other usages elsewhere

Reply:
I've replaced any lodash implementations of startsWith to use standard lib startsWith